templates/base.html

```diff
@@ -29,29 +29,33 @@
           <li>
             <a class="admin" href="{% url 'admin:index' %}">
               <i class="fa fa-wrench" aria-hidden="true"></i>
+              {% trans "Django admin" %}
```

Comment:
`s/Django admin/Admin`

Reply:
I only capitalized the first word of each title.

Reply:
No, I mean remove `Django`, just let `Admin`

Reply:
haha I was thinking about that when your comment appeared 🌟